Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryPersonEntity.kt

```diff
@@ -0,0 +1,14 @@
+package de.rki.coronawarnapp.contactdiary.storage.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import de.rki.coronawarnapp.contactdiary.model.ContactDiaryPerson
+
+@Entity
```

Comment:
Prefer explicit naming of entities and colums via annotation to prevent accidentally breaking the DB by refactoring or angering R8.

Reply:
adjustend all Entities.